src/spikeinterface/curation/mergeunitssorting.py

```diff
@@ -5,7 +5,7 @@
 
 class MergeUnitsSorting(BaseSorting):
     """
-    Class that handles several merges of units from a Sorting object based on a list of list of unit_ids.
+    Class that handles several merges of units from a Sorting object based on a list of lists of unit_ids.
```

Comment:
I still find this confusing
```suggestion
    Class that handles several merges of units from a Sorting object based on a list whose elements are lists of unit_ids.
```
What do you think?

Reply:
I think list of lists is clear enough :)